src/main/kotlin/org/jetbrains/research/transformations/Anonymization.kt

```diff
@@ -0,0 +1,75 @@
+@file:Suppress("UNCHECKED_CAST")
+
+package org.jetbrains.research.transformations
+
+import com.github.gumtreediff.tree.ITree
+import com.github.gumtreediff.tree.Tree
+import com.github.gumtreediff.tree.TreeContext
+
+object Anonymization: Transformation {
+    override val metadataKey = "anonymization"
+    private var anonymNamesMap: MutableMap<String, String> = mutableMapOf()
+    private var numOfVariables = 0
+    private const val VAR_PREFIX = "v"
```

Comment:
Maybe should we add the different prefixes for the different types of nodes? For example, for variable - this prefix, for function name - another one and etc?

Reply:
I agree and I will add different prefixes for different anonymizing essences